File: Waterweigher.py
# repeat is set to 'yes' for while loop to start
repeat = 'yes'

while repeat.lower() == 'yes':
    # asks user for amount of gallons
    gal = float(input("Enter the amount of water in gallons: "))

    # converts gallons to pounds
    weight = gal * 8.34

    # prints the weight of the water in pounds
    print(str(weight))

    # asks user to repeat
    repeat = input("Enter 'yes' if you would like to repeat: ")

File: Java/Waterweigher.java
import java.util.Scanner;

public class Waterweigher {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		double gallons, pounds;

		System.out.print("Enter amound of gallons of water: ");
		gallons = scanner.nextDouble();
		pounds = 8.33 * gallons;
		System.out.println("You have " + pounds + " pounds");
	}
}
